index.html

```diff
@@ -10,8 +10,113 @@
+    <script src="http://www.google.com/jsapi" type="text/javascript"></script>
+    <script type="text/javascript">google.load("jquery", "1.3.2");</script>
   </head>
   <body id="toplevel">
+    <div id="loading-animation-container" class="animate-out">
+    <!-- CREDITS: Loading Circle Animation from https://codepen.io/luka-butina/pen/ZbeQzd -->
+     <div id="loading-content">
+        ::before
+        ::after
+     </div>
+     <style type="text/css">
+        #loading-animation-container {
+          position: fixed;
+          width: 100%;
+          height: 100%;
+          left: 0;
+          top: 0;
+          background: #000000;
+          z-index: 10;
+        }
+
+        #loading-content {
+          display: block;
+          position: relative;
+          left: 50%;
+          top: 50%;
+          width: 150px;
+          height: 150px;
+          margin: -75px 0 0 -75px;
+          border: 3px solid transparent;
+          border-top-color: #fff;
+          border-bottom-color: #fff;
+          border-radius: 50%;
+          animation: loader 2s linear infinite;
+        }
+
+        #loading-content:after {
+          content: "";
+          position: absolute;
+          left: 15px;
+          right: 15px;
+          top: 15px;
+          bottom: 15px;
+          border: 3px solid transparent;
+          border-top-color: #fff;
+          border-bottom-color: #fff;
+          border-radius: 50%;
+          animation: loader 1.5s linear infinite;
+        }
+
+        #loading-content:before {
+          content: "";
+          position: absolute;
+          left: 5px;
+          right: 5px;
+          top: 5px;
+          bottom: 5px;
+          border: 3px solid transparent;
+          border-top-color: #fff;
+          border-bottom-color: #fff;
+          border-radius: 50%;
+          animation: loader 3s linear infinite;
+        }
+
+        @-webkit-keyframes loader {
+          0% {
+            transform: rotate(0deg);
+          }
+          100% {
+            transform: rotate(360deg);
+          }
+        }
+
+        @keyframes loader {
+          0% {
+            transform: rotate(0deg);
+          }
+          100% {
+            transform: rotate(360deg);
+          }
+        }
+
+        .animate-out {
+            animation-name: fadeFromTop;
+            animation-duration: 400ms;
+            animation-timing-function: ease-in;
+            animation-fill-mode: forwards;
+        }
+
+        @keyframes fadeFromTop  {
+            0%{
+                opacity:1;
+                transform: translateX(0px);
+            }
+            100%{
+                opacity: 1;
+                transform: translateX(-100%);
+            }
+        }
+     </style>
+     <script type="text/javascript">
+      var hideOverlay = function(){
+            overlay = document.getElementById("loading-animation-container");
+            overlay.classList.add("animate-out");
+        }
+     </script>
+</div>
```

Comment:
Indenation error persisting from previous PR. Please use an editor like Sublime with a plugin like HTML Prettify to fix this.